tree/dataframe/test/dataframe_snapshot_ntuple.cxx

```diff
@@ -35,6 +35,15 @@ class FileRAII {
    std::string GetPath() const { return fPath; }
 };
 
+template <typename T>
+void expect_vec_eq(const ROOT::RVec<T> &v1, const ROOT::RVec<T> &v2)
+{
+   ASSERT_EQ(v1.size(), v2.size()) << "Vectors 'v1' and 'v2' are of unequal length";
+   for (std::size_t i = 0ull; i < v1.size(); ++i) {
+      EXPECT_EQ(v1[i], v2[i]) << "Vectors 'v1' and 'v2' differ at index " << i;
```

Comment:
Just a heads up that we may need `EXPECT_FLOAT_EQ` here, depending on what `T` is. I think for the moment it's fine because `{ 1.f, 2.f, 3.f }` are exactly representable.